tests/images/e2e/Dockerfile

```diff
@@ -19,4 +19,5 @@ ADD tidb-cluster /charts/e2e/tidb-cluster
 ADD tidb-backup /charts/e2e/tidb-backup
 ADD manifests /manifests/e2e
 
-ADD bin/e2e /usr/local/bin/e2e
+ADD bin/ginkgo /usr/local/bin/
```

Comment:
Where does this binary come from?

Reply:
built by `make e2e-build`

Reply:
Oh, I see. But do we need a specific ginkgo version? When enabling GO111MODULE, the go.sum will be changed if we always pull the latest ginkgo.

Reply:
I locked the ginkgo version (see go.mod changes). `go build <full-path-library>` in a go modules project will build the version in `go.mod` file. (will try to fetch the latest only if the version is missing)